stubs/openpyxl/openpyxl/packaging/core.pyi

```diff
@@ -21,38 +22,38 @@ class QualifiedDateTime(NestedDateTime):
 class DocumentProperties(Serialisable):
     tagname: str
     namespace: Incomplete
-    category: Incomplete
-    contentStatus: Incomplete
-    keywords: Incomplete
-    lastModifiedBy: Incomplete
+    category: NestedText[str, Literal[True]]
+    contentStatus: NestedText[str, Literal[True]]
+    keywords: NestedText[str, Literal[True]]
+    lastModifiedBy: NestedText[str, Literal[True]]
     lastPrinted: Incomplete
-    revision: Incomplete
-    version: Incomplete
+    revision: NestedText[str, Literal[True]]
+    version: NestedText[str, Literal[True]]
     last_modified_by: Alias
-    subject: Incomplete
-    title: Incomplete
-    creator: Incomplete
-    description: Incomplete
-    identifier: Incomplete
-    language: Incomplete
+    subject: NestedText[str, Literal[True]]
+    title: NestedText[str, Literal[True]]
+    creator: NestedText[str, Literal[True]]
+    description: NestedText[str, Literal[True]]
+    identifier: NestedText[str, Literal[True]]
+    language: NestedText[str, Literal[True]]
     created: Incomplete
     modified: Incomplete
     __elements__: ClassVar[tuple[str, ...]]
     def __init__(
         self,
-        category: Incomplete | None = None,
-        contentStatus: Incomplete | None = None,
-        keywords: Incomplete | None = None,
-        lastModifiedBy: Incomplete | None = None,
+        category: object = None,
+        contentStatus: object = None,
+        keywords: object = None,
+        lastModifiedBy: object = None,
         lastPrinted: Incomplete | None = None,
-        revision: Incomplete | None = None,
-        version: Incomplete | None = None,
+        revision: object = None,
+        version: object = None,
         created=None,
-        creator: str = "openpyxl",
-        description: Incomplete | None = None,
-        identifier: Incomplete | None = None,
-        language: Incomplete | None = None,
+        creator: object = "openpyxl",
+        description: object = None,
+        identifier: object = None,
+        language: object = None,
         modified=None,
-        subject: Incomplete | None = None,
-        title: Incomplete | None = None,
+        subject: object = None,
+        title: object = None,
```

Comment:
You have converted some `Incomplete | None` annotations to `object`. (Here and elsewhere.) I didn't check with the implementation, but it seems to me that supplying arbitrary objects to, e.g., `title` can't be right.

Reply:
There are "String Convertible" descriptors that converts whatever value you give to a string, so they can really be given anything.

"Boolean convertible" do have some intended values (even if in theory you could pass anything, same reason we type params as boolean even if the truthyness is checked). But I haven't found such intended values for "String Convertible"